compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -1093,6 +1093,18 @@ def IREELinalgExt_WinogradInputTransformOp : IREELinalgExt_Op<"winograd.input_tr
     Value output() {
       return getDpsInitOperand(0)->get();
     }
+    Value getOriginalOperand() {
+      return input();
+    }
+    Value getTransformedOperand() {
+      return output();
+    }
+    ShapedType getOriginalOperandType() {
+      return getInputOperandType();
+    }
+    ShapedType getTransformedOperandType() {
+      return getOutputOperandType();
+    }
```

Comment:
Why do we need these methods? Can't we use `input()`, `output()`, `getInputOperandType()` and `getOutputOperandType()` directly?

Reply:
This helps me use some of the same patterns for input and output transforms in the decomposition, since the ops are similar.